plugins/obs-webrtc/whip-output.cpp

```diff
@@ -516,26 +495,43 @@ void WHIPOutput::StopThread(bool signal)
 	last_video_timestamp = 0;
 }
 
-void WHIPOutput::Send(void *data, uintptr_t size, uint64_t duration, int track)
+void WHIPOutput::Send(void *data, uintptr_t size, uint64_t duration,
+		      std::shared_ptr<rtc::Track> track,
+		      std::shared_ptr<rtc::RtcpSrReporter> rtcp_sr_reporter)
 {
-	if (!running)
+	if (track == nullptr || !track->isOpen())
 		return;
 
-	// Sample time is in microseconds, we need to convert it to seconds
+	std::vector<rtc::byte> sample{(rtc::byte *)data,
+				      (rtc::byte *)data + size};
+
+	auto rtp_config = rtcp_sr_reporter->rtpConfig;
+
+	// sample time is in us, we need to convert it to seconds
```

Comment:
I'd prefer the old comment. It always takes me a few seconds to realize that "us" is meant to be "microseconds", and the longer comment doesn't hurt anything.

Reply:
Changed back to old comment